client/my-sites/migrate/step-source-select.jsx

```diff
@@ -116,11 +117,11 @@ class StepSourceSelect extends Component {
 					<CardHeading>{ translate( 'What WordPress site do you want to import?' ) }</CardHeading>
 					<div className="migrate__explain">
 						{ translate(
-							"Enter a URL and we'll help you move your site to WordPress.com. If you already have a" +
+							"Enter a URL and we'll help you move your site to WordPress.com. If you already have a " +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'Enter a URL and we'll help you move your site to WordPress.com. If you already have abackup file, you can {{uploadFileLink}}upload it to import content{{/uploadFileLink}}.' )` *ES Score: 9*